WalletWasabi/Blockchain/Analysis/FeesEstimation/AllFeeEstimate.cs

```diff
@@ -28,7 +28,7 @@ public AllFeeEstimate(EstimateSmartFeeMode type, IDictionary<int, int> estimatio
 			var filteredEstimations = estimations
 				.Where(x => x.Key >= targets[0] && x.Key <= targets[^1])
 				.OrderBy(x => x.Key)
-				.Select(x => (ConfirmationTarget: x.Key, FeeRate: x.Value, Range: targetRanges.First(y => y.Start < x.Key && x.Key <= y.End)))
+				.Select(x => (ConfirmationTarget: x.Key, FeeRate: x.Value, Range: targetRanges.FirstOrDefault(y => y.Start < x.Key && x.Key <= y.End)))
```

Comment:
Isn't this why line 29 exists?

Reply:
If you would return default here, you would return `(0, 0)`? Would that be correct?